assets/js/components/sell-quick-start.component.js

```diff
@@ -100,7 +101,11 @@ function sellQuickStartController ($scope, $rootScope, currency, buySell, Alerts
       this.transaction.fiat = quote.quoteAmount / 100;
     }
     $scope.quote = quote;
-    $scope.status = {};
+    if (this.transaction.btc > $scope.totalBalance) {
+      $scope.status.busy = true;
```

Comment:
Before, this was clearing the whole status object, could not doing that have side effects? Do you think it would be safer to do `$scope.status = { busy: true }` instead?

Also, seems strange to be setting `busy = true` in a success callback, what is this for?